core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -7469,7 +7469,14 @@ RelExpr *Scan::bindNode(BindWA *bindWA)
      bindWA->setErrStatus();
      return NULL;
   }
-
+  if (naTable->hasLobColumn() && isSampleScan())
```

Comment:
Does this mean we cannot update statistics with sampling on a table with LOB columns?